src/views/Joseki/Joseki.tsx

```diff
@@ -297,6 +299,11 @@ class _Joseki extends React.Component<JosekiProps, JosekiState> {
             pgettext("This is a Joseki Tag", "White gets the centre and right");
             pgettext("This is a Joseki Tag", "Black gets sente");
             pgettext("This is a Joseki Tag", "White gets sente");
+
+            // and move categories
+            Object.keys(MoveCategory).forEach((category) => {
+                pgettext("Joseki move category", category);
```

Comment:
I'm fairly certain you need to specify all the string literals here (or elsewhere) in order for the scripts to know what phrases to hand off to the translators.